### ISSUE TITLE
integrate pyedf into mne

### ISSUE THREAD

**Issue by skjerns** [bids-standard/pyedf#9]
Would it be an idea to integrate your code into [MNE](https://github.com/mne-tools/mne-python)?

Currently `mne` lacks an interface to export to EDF, which is quite a drawback.

As your solution is pure-python it could be easily implemented into MNE. Would it be okay to use your code to do so?

related : https://github.com/mne-tools/mne-python/issues/5755

**Comment by palday** [comment]
MNE can't export to *any* format besides its native FIFF (well, and to a pandas DataFrame but that doesn't really count). Note also that several MNE contributors are involved with pyedf and pybv -- at least for me, I haven't copied my contributions (which form the basis of pybv) over to MNE because it adds maintainance overhead while making relatively little difference in usability. It's relatively straighforward to use the `get_data()` method of most MNE classes to get a NumPy array that you can then pass to pyedf and pybv. For pybv, my original work is in [`philistine`](https://philistine.readthedocs.io/en/latest/api/philistine.mne.write_raw_brainvision.html#philistine.mne.write_raw_brainvision) which operates directly one MNE `Raw` objects.

In other words: within the contraints of the relevant licenses, there's nothing stopping you from opening a PR over at MNE-Python. I'm not sure it's worth the effort though and suspect that I may not be the only dev with this opinion. 

If you really want to add this functionality to MNE, the best solution would be to add a dependency on pyedf and pybv and then just have a thin wrapper in MNE that extracts the metadata `.info` and the NumPy array `.get_data()` and passes those over to the relevant functions in pyedf and pybv. It's been on my todo list for a while now to rewrite `philistine.mne.write_raw_brainvision` this way, but it's a low priority compared to all the other contributions I've promised MNE and other projects.

**Comment by skjerns** [comment]
thank's! sounds reasonable.
